mercator-domain-replacements.php

```diff
@@ -44,6 +44,20 @@ public function __construct() {
 			return;
 		}
 
+		// For FACET WP only replace before
```

Comment:
A déplacer dans une méthide à part, puis faire un "true === $this->is_facet_wp()" et ajouter le hook dans le if

Reply:
Done